ui/app/rif/pages/domainsDetailPage/domainDetailActive/domainDetailActive.js

```diff
@@ -0,0 +1,197 @@
+		Object.keys(cryptos).forEach(function(key) {
+			let crypto = cryptos[key]
+			let network = {
+				value: key,
+				label: crypto.name,
+				icon: crypto.icon,
+				color: crypto.color,
+			}
+			networks.push(network)
+		});
+		this.state = { 
+			resolvers: resolvers,
+			selectedResolverIndex: 0,
+			networks: networks,
+			selectedNetwork: networks[0].value,
+			insertedAddress: '',
+		}
+	}
+	_updateNetwork = (selectedOption) => {
+		this.setState({ selectedNetwork: selectedOption.value })
+	}
+	_updateAddress = (address) => {
+		this.setState({ insertedAddress: address })
+	}
+	_addAddress = () => {
+		let domains = JSON.parse(localStorage.rnsDomains)
+		let myIndex = -1
+		let resolverIndex = this.state.selectedResolverIndex
+		let selecteddomain = domains.find((domain, index) => {
+			if(domain.domain === this.props.domain.domain){
+				myIndex = index
+				return domain
+			}
+		})
+
+		if(myIndex !== -1 && resolverIndex !== -1){
+			let newNetwork = {
+				networkName: this.state.selectedNetwork,
+				networkIcon:this.state.selectedNetwork,
+				address: this.state.insertedAddress,
+			}
+			domains[myIndex].resolvers[resolverIndex].network.push(newNetwork)
+			localStorage.setItem('rnsDomains', JSON.stringify(domains))
+			//Sending back with localstorage rnsDomains (Here we try to get again localstorage so if it wasnt updated, we're going to show whats really saved)
+			domains = JSON.parse(localStorage.rnsDomains)
+			this.setState({
+				resolvers: domains[myIndex].resolvers
+			})
+		}
+	}
+	showModalAddNetworkAddress = () => {
+		let elements = []
+		elements.push(<AddNewTokenNetworkAddress 
+			updateNetwork={this._updateNetwork.bind(this)}
+			updateAddress={this._updateAddress.bind(this)}
+			networks={this.state.networks}
+		/>)
+		let message = {
+			title: 'Add new network',
+			body: { 
+				elements: elements
+			},
+			confirmLabel: 'SAVE',
+			cancelLabel: 'CANCEL',
+			confirmCallback: () => {
+				this._addAddress()
+			},
+			cancelCallback: () => {
+			},
+		}
+		this.props.addNewNetwork(message)
+	}
+	render () {
+		const { domainName, address, content, expirationDate, autoRenew, ownerAddress, isOwner, isLuminoNode, isRifStorage } = this.props
+		let networks = this.state.resolvers[this.state.selectedResolverIndex].network.map((network, index) => {
+			return <div key={index} className={'resolver-network-description'}>
+					<FontAwesomeIcon icon={getIconForToken(network.networkIcon).icon} color={getIconForToken(network.networkIcon).color} className={'domain-icon'}/>
+					<span>{network.networkName}</span>
+					<span className={'resolver-network-description-address'}>{network.address}</span>
+				</div>
+			})
+		return (
+		<div className={'body'}>
+            <div id='headerName' className={'domain-name'}>
+                <span>{domainName}</span>
+                {isOwner &&
+                    <FontAwesomeIcon icon={faCheckCircle} color="#000080" className={'domain-icon'}/>
```

Comment:
I suggest to put these color values in constants, maybe an object with the colors inside.

Reply:
I think it should be components. 

Like the LuminoIcon, StorageIcon...

In a future there will be on click actions, and probably we are going to use those icons with actions in another pages. So it makes sense to make a component

Reply:
All this icons were moved to components now